### ISSUE TITLE
Russian/Polish tooltip strings out of bounds

### ISSUE THREAD

**Issue by smunteankestra** [kestra-io/kestra#4637]
### Describe the issue

![image](https://github.com/user-attachments/assets/fb709cb7-7d6a-4508-8e34-6bffa184c39b)

![image](https://github.com/user-attachments/assets/8da27481-20c3-46f4-b192-d86c6c385273)

### Environment

- Kestra Version: develop
- Operating System (Docker):
- Java Version ( Docker):

**Comment by aninditamitro** [comment]
@smunteankestra I would like to contribute to this issue. Please assign it to me if no one else is looking into it.

**Comment by MilosPaunovic** [comment]
@aninditamitro It would be really beneficial if you could check all other available languages, as well. 

in any case, thanks for your will to contribute, it's greatly appreciated! 🚀

**Comment by aninditamitro** [comment]
@MilosPaunovic  Thank you for your feedback and kind words. I will take the time to check the other available languages and ensure I provide valuable input. I appreciate the opportunity to contribute to the project.

**Comment by Ben8t** [comment]
@aninditamitro did you get the chance to move forward on this issue ?

**Comment by aninditamitro** [comment]
> @aninditamitro did you get the chance to move forward on this issue ?

Thank you for checking in. I’ve reviewed the documentation and have gained a good understanding of the issue. However, I’m currently unsure of the best approach to solve it. Could you provide some guidance on how I should proceed or suggest any resources that might be helpful in tackling this?

I appreciate your support and look forward to your suggestions.

**Comment by MilosPaunovic** [comment]
You would need to locate all the strings that go out of bounds on languages other than English and find a way to show trimmed text (ellipsis) with hover tooltip which show the entire string.

**Comment by MilosPaunovic** [comment]
Is there any progress with this @aninditamitro?

**Comment by kendraT12** [comment]
Hi, my team and I would like to work on this.

The other team members are:
@dessources 
@Mehruza

**Comment by MilosPaunovic** [comment]
Alright, go for it @kendraT12, and we'll wait for a PR! 🚀

**Comment by dessources** [comment]
Hi @MilosPaunovic , we looked at the issue but were unable to reproduce it. 
We noticed that the UI of the execution pane -Which is where the issue was indicated to be - is different today then the way it was back when this issue was first posted. Additionally, in the current version it does not seem like the filters like "state" or "tags" are translated when we switch the language to either Russian or Polish, which prevents the issue in the first place. 
This issue might be irrelevant for the project's current version. Please advise

UI screenshot provided in the issue, the filters are separate fields
<img width="1605" height="813" alt="Image" src="https://github.com/user-attachments/assets/9dacbd5a-692f-4a2b-a18b-6febd2732089" />

Current ui, the filters are all inside one big input field

<img width="1905" height="901" alt="Image" src="https://github.com/user-attachments/assets/eacc3c8c-fe1e-4b67-8957-ec1a6e6626a8" />

**Comment by MilosPaunovic** [comment]
Yes, you are absolutely correct there @dessources, thanks for guiding us to this conclusion! 🚀 

If you're still willing to contribute, may I suggest that you check our [curated list of good first issues](https://go.kestra.io/contributing) to see if there is something you find interesting enough and think you can tackle independently. It can be further filtered by desired labels to narrow down the results (suggesting to filter by either `area/backend` or `area/frontend` ones, based on your preferences).
